### PR TITLE
Modify DB seed.

### PR DESCRIPTION
**Pull request by andela-oojewale**
Add necessary validation to some models.
Add more resources to db seed.

### CODE REVIEW

File: db/helpers/seed_helper.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+class SeedHelper
+  def objectify(model)
+    return model.titleize.constantize unless model.include? "_"
```
Comment (andela-echigbo) [inline db/helpers/seed_helper.rb]: 👍  This is not a problem, though, but I believe the next line alone can take care of both conditions.

Comment (andela-oojewale) [inline db/helpers/seed_helper.rb]: Can you give it a try in rails console and let m know what you get? Could be helpful.

Comment (andela-oojewale) [inline db/helpers/seed_helper.rb]: Cool. it worked. Thanks.

Comment (andela-echigbo) [inline db/helpers/seed_helper.rb]: I just tested it out now. It works well. But I could only test with strings that have no `_` as I don't know any of our models that contain that.

Comment (andela-oakinniranye) [inline db/helpers/seed_helper.rb]: I don't think you'd need the `split&map`, you could use ActiveSupport::Inflector `.classify` or `.camelize`.

File: spec/factories/comments.rb [hunk 2]
```diff
@@ -1,12 +1,9 @@
 FactoryGirl.define do
   factory :comment do
     content { Faker::Hacker.say_something_smart }
+    association :comment_on, factory: :question
```
Comment (andela-ooranagwa) [inline spec/factories/comments.rb]: 👍

**Comment by andela-ooranagwa** [comment]
👍

File: db/helpers/seed_helper.rb (hunk 1 above)
Comment (andela-oojewale) [inline db/helpers/seed_helper.rb]: I already switched to `.camelize` before i saw your comment. Thanks.

**Pull request merged**